src/latex/LaTeXWriter.jl

```diff
@@ -83,6 +83,7 @@ Context(io, doc) = Context{typeof(io)}(io, false, Dict(), 1, "", doc)
 
 _print(c::Context, args...) = Base.print(c.io, args...)
 _println(c::Context, args...) = Base.println(c.io, args...)
+_print(io, args...) = Base.print(io, args...)
```

Comment:
I guess the single-argument `latexesc(s)` method wasn't actually tested, because it hit this method error.